test/test_other.py

```diff
@@ -11803,12 +11803,14 @@ def test_split_main_module(self):
     self.assertIn('Hello from lib!', result)
 
   def test_gen_struct_info(self):
-    # This tests is fragile and will need updating any time any of the referenced
-    # structs or defines change.   However its easy to rebaseline with
-    # --rebaseline and it prevents regressions or unintended changes
-    # to the output json.
+    # This test will start failing whenever the struct info changes (e.g. offset or defines
+    # change).  However its easy to rebaseline with --rebaseline.
     self.run_process([PYTHON, path_from_root('tools/gen_struct_info.py'), '-o', 'out.json'])
```

Comment:
Wait, how does rebaseline work? Doesn't the code here need to use this pattern?
```python
if common.EMTEST_REBASELINE:
  # generate the file
```

Reply:
`assertFileContents` automatically takes care of that.

Reply:
Oh, cool!